cc/algorithms/BUILD

```diff
@@ -377,7 +377,6 @@ cc_test(
 
 cc_library(
     name = "numerical-mechanisms-testing",
-    testonly = 1,
```

Comment:
If you remove the testonly attribute, then anyone is free to use this code in a non-test system, which would break DP. Instead, make a second cc_library rule that isn't testonly, and is only visible to your code.